scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -192,21 +192,21 @@ export class LaneCreateCmd implements Command {
   ];
   description = `creates a new lane and switches to it`;
   extendedDescription = `a lane created from main (default-lane) is empty until components are snapped.
-a lane created from another lane has all the components of the original lane.`;
+a lane created from another lane contains all the components of the original lane.`;
   alias = '';
   options = [
     [
       's',
       'scope <scope-name>',
-      'remote scope where this lane will be exported to, default to the defaultScope (can be changed later with "bit lane change-scope")',
+      'remote scope to which this lane will be exported, default to the workspace.json\'s defaultScope (can be changed up to first export of the lane with "bit lane change-scope")',
     ],
     ['', 'remote-scope <scope-name>', 'DEPRECATED. use --scope'],
     [
       '',
       'alias <name>',
       'a local alias to refer to this lane, defaults to the `<lane-name>` (can be added later with "bit lane alias")',
     ],
-    ['', 'fork-lane-new-scope', 'allow forking a lane into a different scope than the original lane'],
+    ['', 'fork-lane-new-scope', 'allow forking a lane into a different scope to the original lane'],
```

Comment:
yep i got it, think the word `to` is misleading here like you said (it doesnt mean 'into' here, it means "different.. to"
though the wording 'fork' here is also, as it could be a `bit fork` or a conceptual lane fork

How about "create the new lane in a different scope than its parent lane (if created from another lane)"?

Reply:
sounds good. I'd just replace the word "parent" with "original" (that's how elsewhere we refer to that lane)

Reply:
hmm, i dont think original is a good word here. It's not obvious what you mean, as this isn't a copy of something, you're branching off from some**where**, which is the parent lane of this lane.
or another better term, but 'original' is misleading
(which means we may need to change it elsewhere too)

Reply:
agreed. ok great, so let's go with parent and update as we go along as you said